docs/ForkClone.md

```diff
@@ -0,0 +1,57 @@
+# Fork and Clone 
+
+## Steps to fork the repo: 
+
+### Step 1
+
+Go to the main GitHub page of the repo:
+
+(https://github.com/gbowne1/codebooker/)
+
+### Step 2
+
+Once you are on the main GitHub page of the repo, click the Fork button at the top right of the page. 
+
+![](fork.png)
+
+### Step 3
+
+You will now be on a page with the heading "Create a new fork".
+There are two fields which are required, Owner and Repository name. Make sure the owner is you and the repository name is correct. These fields should already be populated when you cliked to this page in Step 2.
+
+![](CreateFork.png)
+
+### Step 3
+
+Click on the green "Create fork" button on the lower right hand corner of the page
+
+![](CreateForkBtn.png)
+
+### Step 4
+
+You will now have a forked version of the original repo that is up to date with the master branch of the original repo.
+
+![](ForkedBranch.png)
+
+## Steps to clone the repo:
+
+### Step 1
+
+Clone this repo to your local machine using (https://github.com/gbowne1/codebooker.git)
```

Comment:
I think, on this place we need to clone the forked repo.Not the main repo

Reply:
One could definitely either clone or fork.  They are similar but do different things: 

<https://stackoverflow.com/questions/7057194/what-is-the-difference-between-forking-and-cloning-on-github>

Reply:
Yeah, But I thing these steps are for cloning the forked project ((i.e) the continuation of the forked project🤷‍♂️